retrofit2-clients/src/main/java/com/palantir/remoting3/retrofit2/Retrofit2ClientBuilder.java

```diff
@@ -108,12 +114,7 @@ private OkHttpClient createOkHttpClient(String userAgent, List<String> uris) {
         // increase default connection pool from 5 @ 5 minutes to 100 @ 10 minutes
         client.connectionPool(new ConnectionPool(100, 10, TimeUnit.MINUTES));
 
-        Dispatcher dispatcher = new Dispatcher();
-
-        dispatcher.setMaxRequests(256);
-        dispatcher.setMaxRequestsPerHost(256);
-
-        client.dispatcher(dispatcher);
+        client.dispatcher(createDispatcher());
```

Comment:
This is only used for the async flow, which is never used with the JAX-RS clients.